RLTS/SendEmail/etc/frontend/di.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\Framework\App\RouterList">
+        <arguments>
+            <argument name="routerList" xsi:type="array">
+                <item name="emailRouting" xsi:type="array">
+                    <item name="class" xsi:type="string">RLTS\SendEmail\Controller\Router</item>
+                    <item name="disable" xsi:type="boolean">false</item>
+                    <item name="sortOrder" xsi:type="string">40</item>
+                </item>
+            </argument>
+        </arguments>
+    </type>
+    <type name="RLTS\SendEmail\Logger\Handler">
```

Comment:
Overcomplicated code - the code around setting custom file name / logger is unnecessary overcomplicated. See the https://drive.google.com/file/d/1ed3KeOMXJoXriWg7CJeEh4rEGzFkU1vk/view?usp=share_link . There is no need to write any separate logger/handler files.